src/daemon/ca/manager.rs

```diff
@@ -321,11 +328,55 @@ impl CaManager {
         self.ca_store.has(handle).map_err(Error::AggregateStoreError)
     }
 
-    // Delete a CA after revocations and withdrawals.
-    pub async fn delete_ca(&self, handle: &Handle, actor: &Actor) -> KrillResult<()> {
-        warn!("Deleting CA '{}' as requested by: {}", handle, actor);
-        self.ca_store.drop_aggregate(handle)?;
-        self.locks.drop_ca(handle).await;
+    /// Delete a CA. Let it do best effort revocation requests and withdraw
+    /// all its objects first. Note that any children of this CA will be left
+    /// orphaned, and they will only learn of this sad fact when they choose
+    /// to call home.
+    pub async fn delete_ca(&self, ca_handle: &Handle, actor: &Actor) -> KrillResult<()> {
```

Comment:
How do these changes relate to removal of support for 'embedded'?

Reply:
This was moved down because `CaPublisher` was no longer needed. That type was doing things in `CaManager` *and*  `RepoManager` for embedded repos. Now this can just all be done here.